input/chainsync/plugin.go

```diff
@@ -74,14 +82,27 @@ func init() {
 					DefaultValue: false,
 					Dest:         &(cmdlineOptions.ntcTcp),
 				},
+				{
+					Name:         "bulk-mode",
+					Type:         plugin.PluginOptionTypeBool,
+					Description:  "use the 'bulk' sync mode with NtN (node-to-node). This should only be used against your own nodes for resource usage reasons",
+					DefaultValue: false,
+					Dest:         &(cmdlineOptions.bulkMode),
+				},
 				{
 					Name:         "intersect-tip",
 					Type:         plugin.PluginOptionTypeBool,
 					Description:  "start syncing at the chain tip (defaults to chain genesis)",
 					DefaultValue: true,
 					Dest:         &(cmdlineOptions.intersectTip),
 				},
-				// TODO: intersect-point
+				{
+					Name:         "intersect-point",
+					Type:         plugin.PluginOptionTypeString,
+					Description:  "start syncing at the specified chain point in '<slot>,<hash>' format",
```

Comment:
Can we adjust this so it's `<slot>.<hash>` to match other tools?

Reply:
I updated the format and added the ability to specify multiple intersect points (separated by commas)